cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -163,16 +168,18 @@ class Content extends TContent {
 }
 
 private class FieldContent extends Content, TFieldContent {
-  Field f;
+  IntValue startBitOffset;
+  IntValue endBitOffset;
 
-  FieldContent() { this = TFieldContent(f) }
+  FieldContent() { this = TFieldContent(startBitOffset, endBitOffset) }
 
-  Field getField() { result = f }
-
-  override string toString() { result = f.toString() }
+  override string toString() {
+    result = "[" + startBitOffset.toString() + ".." + endBitOffset.toString() + ")"
```

Comment:
These strings show up in path explanations, and it would be much less helpful for users if they aren't field names. At least I'm fairly sure they'd show up. The Jenkins job for this PR doesn't show any diff for `ir-path-flow.expected`, but is that just a quirk of how the test are scheduled by `make`?

Reply:
You're right (I think they're shown as part of the access path). It should be possible to relate the offsets to a field if we add a third column to `TFieldContent` with a `Type`, and get the old `toString` behavior.

Reply:
9de570b300dd0cdbe336549d79a4b7b1f1197c99 now implements a `toString` that should be equal to the version currently on `main`.